src/dawn_native/metal/DeviceMTL.mm

```diff
@@ -65,10 +65,7 @@ dawnDevice CreateDevice(id<MTLDevice> metalDevice) {
         [mPendingCommands release];
         mPendingCommands = nil;
 
-        delete mMapTracker;
         mMapTracker = nullptr;
```

Comment:
These one I keep because the device is freed explicitly. When we spend more time on the Metal backend we might want to add an RAII ObjC smart pointer.